app/javascript/react/screens/App/Overview/components/ModalWizard/ModalWizardBody.js

```diff
@@ -17,9 +17,9 @@ class ModalWizardBody extends React.Component {
     bindMethods(this, ['onStepClick', 'stepProps', 'renderLoading']);
   }
 
-  onStepClick(stepIndex) {
+  onStepClick(stepIndex, disabled) {
     const { steps, goToStep, disableNextStep } = this.props;
-    if (disableNextStep) return;
+    if (disableNextStep || disabled) return;
```

Comment:
I think you can pull in `this.props.activeStepIndex` here, and make this line into:
```
if (disableNextStep || stepIndex === activeStepIndex + 1 || disabled) return;
```

That way if the Next button works, the next number button also works.